### DISCUSSION TITLE
Suggestion to Add A Feature To Hide Selected Audio Programs

### DISCUSSION THREAD

**Discussion by Home-Rowed** [File-New-Project/EarTrumpet#1153] [Ideas]
I recently found out about your program, and love it. And I'd like to suggest that a feature to allow hiding user selected programs from the list of shown audio emitting applications. They could be sectioned off into a small collapsed button in one of the corners or in a new right click menu item to show hidden programs.

In my case, I have System Sounds, Discord, and for some reason svchost, that I have no need to ever adjust the volume or mute, so removing those from the list would reduce the UI size, and clutter.

**Comment by Nsbx** [comment]
I confirm this would be awesome
![image](https://user-images.githubusercontent.com/8930930/168417694-72188cec-a757-4130-b4ca-4587adbf9375.png)
I have program like Email client, Alfred clone program and screenshot program showed in the list, even if they make sound i doesn't need to manage their level

**Reply by fishwithfeet** [reply]
Same here, bunch of apps that never make sounds and they are still there :(
